src/bindings/c/include/openvino/c/ov_common.h

```diff
@@ -181,14 +182,18 @@ typedef enum {
     I32,             //!< i32 element type
     I64,             //!< i64 element type
     U1,              //!< binary element type
+    U2,              //!< u2 element type
+    U3,              //!< u3 element type
     U4,              //!< u4 element type
+    U6,              //!< u6 element type
     U8,              //!< u8 element type
     U16,             //!< u16 element type
     U32,             //!< u32 element type
     U64,             //!< u64 element type
     NF4,             //!< nf4 element type
     F8E4M3,          //!< f8e4m3 element type
     F8E5M3,          //!< f8e5m2 element type
+    STRING,          //!< string element type
```

Comment:
I suppose we need explicit converter like in Java API https://github.com/openvinotoolkit/openvino_contrib/pull/910 to avoid such issues, right?

Reply:
BTW, do we have other enums which rely on the same values for C and C++ ?

Reply:
We have had such converter already, but the new element type still need be added to `enum ov_element_type_e`, which will be exposed to users.

The original issue is that we directly use below code line to convert C++ element type to C, it will cause error if element type definition is not aligned between C++ and C:

```
auto type = (ov::element::Type_t)port->object->get_element_type();
*tensor_type = (ov_element_type_e)type;
```

So we should use the converter to convert them to avoid mismatch issue.

Reply:
> BTW, do we have other enums which rely on the same values for C and C++ ?

Should be no more.